openvpn/tunnelDown.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 /etc/transmission/stop.sh
-[[ ! -f /opt/tinyproxy/stop.sh ]] || /opt/tinyproxy/stop.sh
+[[ ! -f /opt/privoxy/stop.sh ]] || /opt/privoxy/stop.sh
```

Comment:
Same comment here about reversing the condition to make it clearer:
```suggestion
[[ -f /opt/privoxy/stop.sh ]] && /opt/privoxy/stop.sh
```